### PR TITLE
Fix: Mode and Mlflow doc added

### PR DESCRIPTION
**Pull request by MilanBariya**
### Describe your changes :

I worked on the ..... because Fix: #6772 

#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-docs/content/openmetadata/connectors/ml-model/mlflow/index.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+<Requirements />
+
+<MetadataIngestionService connector="Mlflow"/>
+
+<h4>Connection Options</h4>
+
+- **trackingUri**: Mlflow Experiment tracking URI. E.g., http://localhost:5000
+- **registryUri**: Mlflow Model registry backend. E.g., mysql+pymysql://mlflow:password@localhost:3307/experiments
+
+<IngestionScheduleAndDeploy />
```
Comment (ulixius9) [inline openmetadata-docs/content/openmetadata/connectors/ml-model/mlflow/index.md:17]: Here, we will need to build a new component like `MlModelIngestionConfig` just like we did for Databases and other sources

**Review by ulixius9 — COMMENTED** [review]
Added a comment rest of the things looks good

File: openmetadata-docs/content/openmetadata/connectors/ml-model/mlflow/index.md (hunk 1 above)
Comment (ulixius9) [reply openmetadata-docs/content/openmetadata/connectors/ml-model/mlflow/index.md:17]: You can refer to this [pr](https://github.com/open-metadata/docs/pull/155)

**Review by harshach — APPROVED** [review]

**Comment by pmbrull** [comment]
Thanks for the PR @MilanBariya, we can merge it as is

**Pull request merged**